__tests__/app.test.js

```diff
@@ -35,10 +35,39 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
+        expect(body.topics.length).toBe(3);
         body.topics.forEach((topic) => {
           expect(typeof topic.slug).toBe("string");
           expect(typeof topic.description).toBe("string");
         });
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200, responds with an article for the relevant article ID", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article[0].article_id).toBe(2);
+        expect(body.article.length).toBe(1);
+      });
```

Comment:
You should be sending back just the object rather than the array when dealing with singular objects. So no need to do a length check test or access items by index